geaflow/geaflow-plugins/geaflow-store/geaflow-store-rocksdb/src/main/java/com/antgroup/geaflow/store/rocksdb/DynamicRocksdbGraphStoreBase.java

```diff
@@ -35,8 +35,8 @@
 import java.util.List;
 import java.util.Map;
 
-public class GraphMultiVersionedRocksdbStore<K, VV, EV> extends BaseRocksdbGraphStore
-    implements IGraphMultiVersionedStore<K, VV, EV> {
+public class DynamicRocksdbGraphStoreBase<K, VV, EV> extends BaseRocksdbGraphStore
```

Comment:
Naming conventions should remain consistent, DynamicRocksdbGraphStoreBase -> DynamicGraphRocksdbStoreBase

Reply:
issue has been resolved.